misk-crypto/src/test/kotlin/misk.crypto/CiphertextFormatTest.kt

```diff
@@ -0,0 +1,146 @@
+package misk.crypto
+
+import misk.testing.MiskTest
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import java.io.ByteArrayOutputStream
+import java.nio.ByteBuffer
+
+@MiskTest
+class CiphertextFormatTest {
```

Comment:
Not net new code right?

Reply:
yeah just copy pasted but i have to leave the old code while i migrate things with topographer